GitUI/UserControls/RevisionGrid/RevisionGridControl.cs

```diff
@@ -64,6 +64,8 @@ public sealed partial class RevisionGridControl : GitModuleControl
         private readonly BuildServerWatcher _buildServerWatcher;
         private readonly Timer _selectionTimer;
         private readonly GraphColumnProvider _graphColumnProvider;
+        private readonly DataGridViewColumn _messageColumn;
```

Comment:
Yes, it could work like the following. Though it still needs to store a column but more generic.
Did you think of more abstraction?
```C#
        private readonly List<ColumnProvider> _columnProviders = new List<ColumnProvider>();
        private DataGridViewColumn _maximizedColumn = null;

        public void ReloadSettings()
        {
            foreach (var provider in _columnProviders)
            {
                provider.ReloadSettings();
                if (provider.Column.AutoSizeMode == DataGridViewAutoSizeColumnMode.Fill)
                {
                    _maximizedColumn = provider.Column;
                }
            }

            if (_maximizedColumn != null)
            {
                _gridView.Columns.GetLastColumn(DataGridViewElementStates.Visible, DataGridViewElementStates.None)
                    .Resizable = DataGridViewTriState.False;
            }
        }
```

Reply:
I'm not sure whether that's better or not. Feels like there might be something that doesn't require remembering columns, but nothing comes to mind.